config/nrfconnect/chip-module/Kconfig

```diff
@@ -397,6 +397,33 @@ config CHIP_CRYPTO_PSA_DAC_PRIV_KEY_KMU_ENCRYPTED
 
 endif # CHIP_CRYPTO_PSA_MIGRATE_DAC_PRIV_KEY
 
+config CHIP_STORE_KEYS_IN_KMU
+	bool "Use KMU to store crypto keys"
+	default y
+	depends on CRACEN_LIB_KMU
+	help
+	  Uses KMU to store keys if it is available on the device. This option uses
+	  nrfconnect custom crypto implementation of storing keys. All crypto materials
+	  used in Matter stack will be persisted in the KMU slots.
+
+if CHIP_STORE_KEYS_IN_KMU
+	config CHIP_CORE_KMU_SLOT_START
+	int "Initial KMU slot number for Matter keys"
+	default 100
+	help
+	  Starting slot number dedicated for Matter purposes.
```

Comment:
Maybe: `Defines the initial KMU slot number reserved for storing Matter-related keys. This configuration does not include the DAC private key.`